### ISSUE TITLE
Missing backdrop_set_message status messages on several admin pages

### ISSUE THREAD

**Issue by PaulHoza** [backdrop/backdrop-issues#1060]
Form submits on several admin pages do not update the status message.  I'll go through and find more of these (if they exist), but here are the first two pages I noticed missing backdrop_set_message in the submit handlers:

/admin/config/administration/admin-bar
/admin/reports/updates/settings

(Thanks to @serundeputy for helping me in IRC, both to confirm this issue and to quickly send me on a path to fixing it!)

**Comment by MrHaroldA** [comment]
Is this due to the "missing" system_settings_form()?

Modules like Domains also rely on that function to save all domain-specific variables. Too bad it's gone.

**Comment by serundeputy** [comment]
not because of `system_settings_form()`, just needs a `backdrop_set_message` in the submit callback.

~Geoff

**Comment by PaulHoza** [comment]
BTW, I've gotten a good dozen+ of these fixed. I'm trying to find more of them to see how comprehensive I can be about finding more, but it's a little bit tedious and confusing at times since I don't really know the system well enough from the inside.  Hopefully I do at least a moderate job of doing this whole thing (e.g. PR and such with a lot of things to commit), as this blossomed in scope quite a bit from what I expected... as things tend to do. :)

Most of my struggle has been getting a better workflow of working with git/github and setting up a decent process on my end.  I'm shooting for submitting all this "soon".

**Comment by quicksketch** [comment]
Hi @PaulHoza! Thanks for doing this work! Have you managed to get this into a PR yet? I'd love to give it a review. :smile:

Even if you don't find all missing messages, it'd be great to get the ones you've found so far. We can do additional follow-ups as more are found.

**Comment by PaulHoza** [comment]
Hi,
I was indeed delaying because I was worried about finding them all.  I got
a bunch, but then thought I better go through and search a few different
ways or such and haven't been finding time to do that.

I'll go ahead and get a PR of the ones I do have now. Thanks for the
encouragement to not stress over 100%.  That helps a lot, actually.

**Comment by quicksketch** [comment]
Super! We're trying to encourage a commit-frequently culture. So don't worry about making it perfect! With new releases every couple of weeks there's always another opportunity. :smile:

And.... PR. Nice one!

**Comment by quicksketch** [comment]
The PR looks great. Just one line ended up with a couple of extra spaces: https://github.com/backdrop/backdrop/pull/1022/files#r37692673

Otherwise, everything looks great!

**Comment by PaulHoza** [comment]
Whew!  Again, I'm embarrassed at how clunky I feel and how nervous I am about doing this stuff.  I'm really glad it worked again, but I took WAY too long inbetween when I started this work and when I did this PR.  I had to re-learn it all again and did things a bit wonky on my end.. oh well, it worked. 

Thanks again for the encouragement and kindness.  Sorry it took me so long to submit this.

I really need to find a flow with using git, so I'm glad I can at least contribute something useful while I'm learning a better process.

Cheers.
